css/style.css

```diff
@@ -1,4 +1,672 @@
+  .header__logo-img{
+    padding-left: 15px;
+    width: 167px;
+    height: 50px;
+  }
+  .menubar__link {
+    display: none;
+  }
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+  .second {
+    top: 9px;
+  }
+  .third {
+    top: 18px;
+  }
+  .main-tesla {
+    display: flex;
+    justify-content: center;
+  }
+  .tesla-bgd{
+    top: -22px;
+    max-width: 820px;
+    padding-left: 35px;
+  }
+  .tesla-car {
+    max-width: 701px;
+    height: auto;
+    padding: 84px 0 30px 0px;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 58px;
+  }
+  .content__facts {
+    padding: 0px 0px 54px 0px;
+  }
+  .data__speed-block {
+    padding-right: 92px;
+  }
+  .data__speed-icon {
+    width: 78px;
+    height: 62px;
+    padding: 7px 14px 0 0;
+  }
+  .content__road-icon {
+    width: 48px;
+    height: 77px;
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 51px;
+    padding-right: 6px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:  #e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+  .arrow {
+    padding-left: 2px;
+  }
+  
+}
+ @media (min-width:568px) and (max-width:767px) {
+  .menubar__link {
+    display: none;
+  }
+  .content__main-title {
+    font-size: 75px;
+  }
+  .tesla-bgd{
+    top: -22px;
+    max-width: 100%;
+    height: auto;
+    /* max-width: 645px; */
+    padding-left: 22px;
+  }
+  .tesla-car {
+    /* max-width: 600px; */
+    max-width: 100%;
+    height: auto;
+    padding: 61px 0 30px 0px;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 58px;
+  }
+  .content__facts {
+    padding: 0px 0px 54px 0px;
+  }
+  .data__speed-block {
+    padding-right: 92px;
+  }
+  .data__speed-icon {
+    width: 55px;
+    height: 45px;
+    padding: 7px 14px 0 0;
+  }
+  .content__road-icon {
+    width: 40px;
+    height: 58px;
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 43px;
+    padding-right: 6px;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:  #e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)